runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterPlugin.java

```diff
@@ -138,6 +143,25 @@ public void onGameTick(GameTick event)
 		ticksSinceHPRegen = (ticksSinceHPRegen + 1) % ticksPerHPRegen;
 		hitpointsPercentage = ticksSinceHPRegen / (double) ticksPerHPRegen;
 
+		if (config.notifyHitpointsRegen())
+		{
+			if (!hitpointsNotificationSend)
+			{
+				if (hitpointsPercentage == 0.90 && client.getBoostedSkillLevel(Skill.HITPOINTS) == 1)
```

Comment:
https://google.github.io/guava/releases/19.0/api/docs/com/google/common/math/DoubleMath.html#fuzzyEquals(double,%20double,%20double)

https://en.wikipedia.org/wiki/Floating-point_arithmetic#Representable_numbers,_conversion_and_rounding

Consider a tolerance of 1e-6. If 0.1 isn't representable, then I doubt 1 - 0.1 is.